### PR TITLE
Google maps API

### PR DESCRIPTION
**Pull request by adamhundley**
* implemented google maps api
* map loads on properties index page and shows the city searched as well as properties
* needs styled

### CODE REVIEW

**Comment by adamhundley** [comment]
check it son! @Salvi6God

File: Gemfile [hunk 1]
```diff
@@ -21,7 +21,11 @@ gem 'chartkick'
 gem 'groupdate'
 gem "simple_calendar", "~> 2.0"
 gem 'stripe-ruby-mock', '~> 2.2.2', :require => 'stripe_mock'
+<<<<<<< HEAD
```
Comment (Salvi6God) [inline Gemfile]: THAT MERGE CONFLICT THOOOOO!!!!!!!!!!!!!

Comment (adamhundley) [inline Gemfile]: my bad! haha fixed

**Comment by Salvi6God** [comment]
cool dude! I'll merge now

**Pull request merged**